src/App.tsx

```diff
@@ -183,11 +178,22 @@ const getL1ToL2Messages = async (
     })
     if (logFromL2Inbox.length === 0) continue
 
-    // Workaround https://github.com/OffchainLabs/arbitrum-sdk/pull/138
-    if (!l2Network.rpcURL && l2ChainID === 42170) {
-      l2Network.rpcURL = "https://nova.arbitrum.io/rpc"
+    let l2RpcURL
+    switch (l2ChainID) {
+      case 42161: 
+      l2RpcURL = "https://arb1.arbitrum.io/rpc";
+      break;
+      case 42170: 
+      l2RpcURL = "https://nova.arbitrum.io/rpc";
+      break;
+      case 421611: 
+      l2RpcURL = "https://rinkeby.arbitrum.io/rpc";
+      break;
+      case 421613: 
+      l2RpcURL = "https://goerli-rollup.arbitrum.io/rpc";
+      break;
     }
```

Comment:
unknown l2chainid not handled

Reply:
I think it's impossible to have another L2 chain ID. Am I missing sth?
We have just 3 supportedL1Networks (1,4,5). So we'll have just one of these l1networks which is passed to getL1ToL2Messages method. In this method we'll check for partner chain ID. Because the only networks we have on supportedL1Networks is 1,4,5, then we will have one of these L2 networks 42161, 42170, 421611, 421613
Also if user sends tx from another network, they will receive "Error: invalid transaction hash" in the L1 network check.
The only way that user may have a tx hash with a valid L1 and L2 chain that have a withdrawal message is when user adds new network using addCustomNetwork which is not supported in the dashboard at all (and I think we don't need it on dashboard) 
But if you think we need it for testing, etc. I can add default path to supportedL1Networks and also to the l2chainid switch case to use .env to have the L1 and L2 RPC for the custom network.

Reply:
I think best just to add a `default` case and throw an error.